docs/Conformance Criteria/README-Conformance Criteria V2.md

```diff
@@ -0,0 +1,349 @@
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">10</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>v1 Profile Support:</b>Extension has a backwards compatibility and it is able to support v1 type of profiles.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555" rowspan=5>11</th>
+   <th style="background-color:#555555"></th>
+   <th style="background-color:#AAAAAA" colspan=2>Mark (a) or (b) or (c)</th>
+   <th style="background-color:#AAAAAA"></th>
+   <td style="text-align:center">Extension uses the extensibility APIs provided by Zowe Explorer. Supported methods include:<p style="color:red">(Please select all that apply _a_, _b_, or _c_)</td>
+ </tr>
+ <tr>
+   <th style="background-color:#555555"></th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA" ></th>
+   <th></th>
+  <td>a. Extension Accessing Profiles</td>
+ </tr>
+ <tr>
+   <th style="background-color:#555555"></th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA" ></th>
+   <th></th>
+  <td>b. Data Provider Extension</td>
+ </tr>
+ <tr>
+   <th style="background-color:#555555"></th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA" ></th>
+   <th></th>
+  <td>c. Extension Adding Menus</td>
+ </tr>
+</table>
+
+### Extension Accessing Profiles
+
+Criteria for VS Code extensions that want to access the same Zowe CLI profiles that Zowe Explorer uses.
+
+<table rules="all">
+ <thead>
+ <th style=background-color:#5555AA>Item </th>
+ <th style=background-color:#5555AA>Ver </th>
+ <th style=background-color:#5555AA>Required </th>
+ <th style=background-color:#5555AA>Best Practice </th>
+ <th style=background-color:#5555AA>Conformant </th>
+ <th style=background-color:#5555AA>Criteria </th>
+ </thead>
+
+ <tr>
+   <th style="background-color:#555555">12</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>VS Code extension dependency:</b> Extension declares Zowe Explorer as a VS Code extension dependency by including an `extensionDependencies` entry for Zowe Explorer in its package.json file.</td>
+ </tr>
+
+  <tr>
+   <th style="background-color:#555555">13</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Zowe Extender access:</b> Extension accesses the shared Zowe Explorer profiles cache via `ZoweExplorerApi.IApiRegisterClient.getExplorerExtenderApi()` API as documented in the Zowe Explorer extensibility documentation.</td>
```

Comment:
should we also link to the extensibility documentation here?

Reply:
I think it would be good to add the extensibility docs link here.